.github/workflows/sycl_post_commit.yml

```diff
@@ -1,7 +1,8 @@
 name: SYCL Post Commit
 
+
 on:
-  push:
+  pull_request_target:
```

Comment:
@bader @aelovikov-intel, I changed this line to trigger post-commit checks on this PR, but all the checks look skipped, even lint. Is this a correct way to enable post-commit checks now, or probably it currently doesn't work as expected?

Reply:
See https://github.com/intel/llvm/pull/10002 and https://github.com/intel/llvm/pull/10012.

Reply:
Thanks!